packages/flutter/test/material/scaffold_test.dart

```diff
@@ -1840,7 +1840,7 @@ void main() {
         final GlobalKey<ScaffoldState> key = GlobalKey<ScaffoldState>();
         const Key buttonKey = Key('button');
         final List<FlutterErrorDetails> errors = <FlutterErrorDetails>[];
-        FlutterError.onError = (FlutterErrorDetails error) => errors.add(error);
+        FlutterError.presentError = (FlutterErrorDetails error) => errors.add(error);
```

Comment:
This is fairly sad change. The widget tree failed to tear down after this test probably because the widget tree is in a bad shape. The test was passing because it does not restore onError at the end of the test and the error was ignored.